pulpcore/cli/ansible/distribution.py

```diff
@@ -7,14 +7,9 @@
     PulpAnsibleDistributionContext,
     PulpAnsibleRepositoryContext,
 )
-from pulpcore.cli.common.context import (
-    EntityDefinition,
-    EntityFieldDefinition,
-    PulpContext,
-    pass_entity_context,
-    pass_pulp_context,
-)
-from pulpcore.cli.common.generic import PluginRequiredVersion as PRV
+from pulpcore.cli.common.context import EntityDefinition, EntityFieldDefinition
+from pulpcore.cli.common.context import PluginRequiredVersion as PRV
+from pulpcore.cli.common.context import PulpContext, pass_entity_context, pass_pulp_context
```

Comment:
What was bad about the old version of bunched import?

Reply:
Isort automatically did this. It breaks alias imports (`PluginRequiredVersion as PRV`) into a separate line. Since PluginRequiredVersion moved from `pulpcore.cli.common.generic` to `pulpcore.cli.common.context`, it broke up the `pulpcore.cli.common.context` into multiple lines you see here to keep the alphabetic order.

Reply:
I see. Thanks.